lib/about_yml/schema.json

```diff
@@ -142,6 +150,20 @@
                 }
             }
         },
+        "user": {
+            "type": "object",
+            "description": "Organizations or individuals who have adopted the project for their own use",
```

Comment:
I'm wondering if it's worth having some kind of `description` property, so that we could say "team Foo integrated 18F/bar into their Baz product" for more infrastructure-y kinds of projects. Thoughts?

That said, this in and of itself is good to go. Merging.